server/log_rpc_server.go

```diff
@@ -74,18 +75,23 @@ func (t *TrillianLogRPCServer) QueueLeaf(ctx context.Context, req *trillian.Queu
 
 // QueueLeaves submits a batch of leaves to the log for later integration into the underlying tree.
 func (t *TrillianLogRPCServer) QueueLeaves(ctx context.Context, req *trillian.QueueLeavesRequest) (*trillian.QueueLeavesResponse, error) {
-	ctx = util.NewLogContext(ctx, req.LogId)
 	if err := validateQueueLeavesRequest(req); err != nil {
 		return nil, err
 	}
+	logID := req.GetLogId()
```

Comment:
Using accessors has its benefits, like resilient to nil messages and non-nil defaults. Doesn't make much of a difference here, though, so I've reverted back to `req.LogId`.